nuxeo-compound-documents/src/test/java/org/nuxeo/compound/documents/TestCompoundDocumentImporter.java

```diff
@@ -123,4 +124,18 @@ public void testBadCustomCompoundDocumentFolderType() throws IOException {
         assertEquals("Failed to invoke operation " + COMPOUND_FOLDER_DOCTYPE_DETECTION_OPERATION, cause.getMessage());
     }
 
+    @Test
+    public void testCreateNestedCompoundDocuments() throws IOException {
+        Blob blob = CompoundDocumentUtils.getNestedTestArchives();
```

Comment:
Should `getNestedTestArchives` be imported statically to follow the other methods?